app/controllers/moments_controller.rb

```diff
@@ -70,6 +70,18 @@ def destroy
     redirect_to_path(moments_path)
   end
 
+  # POST /moments/1/picture
+  # POST /moments/1/picture.json
+  def picture
+    moment = set_moment(params[:moment_id])
```

Comment:
Can we set the value of moment after line 79?

Reply:
Nice catch :)